metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlEncoder.java

```diff
@@ -353,7 +346,7 @@ else if (!appendLeader(name, value)) {
                     if (value != null) {
                         writeEscaped(value.trim());
                     }
-                    writeTag(Tag.controlfield::close);
+                    writeTag(Tag.controlfield::close, false);
```

Comment:
What does this change do? It seems to be ineffective. And also counterintuitive: Why would a close tag have arguments/attributes?